scripts/contracts.ts

```diff
@@ -160,6 +163,7 @@ export const CONTRACT_FACTORY: Record<ContractName, FactoryContstructor> = {
     PriceOracle: PriceOracle__factory,
     ChildERC20: ChildERC20__factory,
     PolygonDestination: PolygonDestination__factory,
+    RootChainManager: RootChainManager__factory,
```

Comment:
Since we don't need to deploy root chain manager, we can leave it out from here.